README.md

```diff
@@ -7,6 +7,19 @@
 PegaSys Orchestrate is a platform that enables enterprises to easily build secure
 and reliable applications on Ethereum blockchains.
 
+> **Important:** Pegasys Orchestrate Quickstart is only available for Orchestrate customers and
+> PegaSys partners.
+>
+> [Contact PegaSys](https://pegasys.tech/contact/) if you want to purchase Orchestrate or are interested in our partner program.
+>
+> If you are already a partner or customer but we did not provide the credentials to access the Docker registry, request access on [Pegasys Zendesk support site](http://pegasys.zendesk.com/)
+> (create an account with your company email address or login with an existing one) or by sending an email from your work email to
```

Comment:
I'm not sure why the _create an account...._ part is in brackets?  Would this be clearer as: 

To access Zendesk, create an account with your company email or login using an existing Zendesk account. Alternatively, send an email from your company email to support@pegasys.tech